perses/rjmc/geometry.py

```diff
@@ -536,9 +525,9 @@ def _propose_angle(self, angle, beta):
         Bond angle proposal
         """
         theta0 = angle.type.theteq*units.radians
```

Comment:
I think `angle.type.theteq` is actually in degrees.

@swails uses `DEG_TO_RAD` to convert these to radians when he uses these values:
https://github.com/ParmEd/ParmEd/blob/master/parmed/structure.py#L1905